hyperscale/Dockerfile

```diff
@@ -0,0 +1,73 @@
+      org.label-schema.url="https://www.citusdata.com" \
+      org.label-schema.vcs-url="https://github.com/citusdata/citus" \
+      org.label-schema.vendor="Citus Data, Inc." \
+      org.label-schema.version=${VERSION} \
+      org.label-schema.schema-version="1.0"
+
+ENV CITUS_VERSION ${VERSION}.citus-1
+ENV POSTGRES_DB citus
+ENV PGDATA /data
+
+# install Citus and needed extension on hyperscale
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       ca-certificates \
+       curl \
+    && curl -s https://install.citusdata.com/community/deb.sh | bash \
+    && apt-get install -y postgresql-$PG_MAJOR-citus-10.2=$CITUS_VERSION \
+                          postgresql-$PG_MAJOR-hll=2.16.citus-1 \
+                          postgresql-$PG_MAJOR-topn=2.4.0 \
+                          postgresql-14-postgis-3 \
+                          postgresql-plperl-14 \
+                          postgresql-contrib \
+                          postgresql-14-partman \
+                          postgresql-14-tdigest \
+                          postgresql-14-mysql-fdw \
+                          postgresql-14-cron \
+    && apt-get purge -y --auto-remove curl \
+    && rm -rf /var/lib/apt/lists/*
+
+# set shared_preload_libraries to the list of extensions that needs to be loaded when postgresql starts
+RUN echo "shared_preload_libraries='citus,pg_cron,pg_stat_statements'" >> /usr/share/postgresql/postgresql.conf.sample
+
+# set default database name for pg_cron extension
+RUN echo "cron.database_name = 'citus'" >> /usr/share/postgresql/postgresql.conf.sample
+
+# add scripts to run after initdb
+COPY hyperscale/001-db-user-creation.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/002-create_extension_functions.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/003-create_citus_extension.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/004-postgres_replication_origins.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/005-other_extensions.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/006-pg_cron_pg_partman_setup.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/007-grant_usage_fdw.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/008-other_grants.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/009-stat_statements_reset.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/pg_hba.conf /pg_hba.conf
+
+# add health check script
+COPY pg_healthcheck wait-for-manager.sh /
+
+RUN chown -R postgres:postgres /docker-entrypoint-initdb.d
+
+#ARG CITUS_PWD_COMMAND='\\tsed "s\/###CITUS_PASSWORD###\/${CITUS_PASSWORD}\/g" -i \/docker-entrypoint-initdb.d\/001-db-user-creation.sql\n'
+#RUN sed "/# if first arg looks like a flag.*/a ${CITUS_PWD_COMMAND}" -i /usr/local/bin/docker-entrypoint.sh
+#RUN sed "/file_env().*/i export POSTGRES_PASSWORD=$(echo $RANDOM | md5sum | head -c 20;)" -i /usr/local/bin/docker-entrypoint.sh
+
+COPY hyperscale/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
+
```

Comment:
I needed to add `RUN chmod 755 /usr/local/bin/docker-entrypoint.sh` here to make the image work.

Reply:
I was trying to replicate it in my local. You found the problem thanks. I added execute permission into the file in git repository. This will fix it. No need to add additional command on Dockerfile